public/pages/Main/Main.tsx

```diff
@@ -78,7 +79,7 @@ export default class Main extends Component<MainProps, MainState> {
           'ses_account',
           'email_group',
         ],
-        tooltipSupport: serverFeatures.tooltipSupport,
+        tooltipSupport: false,
```

Comment:
Is this a tmp fix?

Reply:
not really, this is when serverFeatures is null, we should assign some boolean to tooltip